frontend/src/metabase-lib/metadata.ts

```diff
@@ -143,6 +144,23 @@ declare function DisplayInfoFn(
   stageIndex: number,
   segment: SegmentMetadata,
 ): SegmentDisplayInfo;
+/**
+ * Even though it seems weird to pass the same query two times,
+ * this function follows the same pattern as the other displayInfo functions.
+ * The first two parameters are always a query, and a stage.
+ * The third parameter is what you would like to have info about.
+ * It just only happens that the thing we're examining is (again) the query itself.
+ *
+ * @example
+ * const query = question.query();
+ * const { isEditable, isNative } = Lib.displayInfo(query, -1, query);
+ *
+ */
+declare function DisplayInfoFn(
+  query: Query,
```

Comment:
I'd still name not required things as `_query` and `_stageIndex` to stress out they are optional

Reply:
It seems that I don't understand something here. How are the initial query and stageIndex optional?

`Lib.displayInfo(query)` doesn't give you anything useful
`Lib.displayInfo(query, -1, query)` returns `{ isNative, isEditable }`

![image](https://github.com/metabase/metabase/assets/31325167/01e4185e-f64c-4819-92d3-dd387b4bf4ab)

Reply:
I meant that you could have in JS/TS a function that accepts one parameter - query - but passes down 3 (query, -1, query)

anyhow, I'd not spend time on it, if @kamilmielnik is agreed, I'd go with a current variant, but first two params with underscore

Reply:
"optional" is not an accurate word here. Their presence is necessary, but they can be anything, i.e.
`Lib.displayInfo(null, undefined, query)` will work.

Reply:
> anyhow, I'd not spend time on it, if @kamilmielnik is agreed, I'd go with a current variant, but first two params with underscore

Yeah, even though it bothers me a lot, we can just proceed with the 3 arguments.

To make it even more obvious that these arguments don't matter, we can type them as `unknown`:
```
declare function DisplayInfoFn(
  _query: unknown,
  _stageIndex: unknown,
  query: Query,
): QueryDisplayInfo;
```

Reply:
first parameter has to be a query seems